lib/tests/unit/agent_test.cc

```diff
@@ -26,6 +26,7 @@ TEST_CASE("Agent::Agent", "[agent]") {
                                                getCaPath(),
                                                getCertPath(),
                                                getKeyPath(),
+                                               getCrlPath(),
```

Comment:
Not sure how easy it is, but can we add a negative test that shows connecting to a cert listed in the CRL is rejected?

Reply:
Added tests for this in cpp-pcp-client. Also added extensive coverage for this in acceptance tests (show that the curl client as well as web-socket connection check CRL and will not connect when a cert has been revoked).